data/coalition/coalition outfits.txt

```diff
@@ -209,6 +209,57 @@ outfit "Fuel Module"
 	description "In their early days of space exploration, the Saryds developed the fuel module to travel far from Saros without worrying about becoming stranded without fuel, or sudden fuel leaks."
 	description "	Nowadays, nearly every system in the Coalition has refuelling facilities, but some captains still find use in the extra jumps these modules offer."
 
+outfit "Decoy Plating"
+	category "Systems"
+	cost 93000
+	thumbnail "outfit/decoy plating"
+	"mass" 7
+	"outfit space" -7
+	"scan interference" 2
+	"radar jamming" 5
+	description "This plating takes advantage of a ship's scan logs, and reproduces the data of other ships nearby, effectively deflecting attempts to scan a ship's true cargo and equipment."
+	description "	Out of a need to protect themselves against the destructive Heliarch torpedoes, the Lunarium also incorporated small, independent radar jamming systems in each section of the plating, if only to act as a last line of defense."
+
+outfit "Refuelling Module"
+	category "Systems"
+	cost 822000
+	thumbnail "outfit/refuelling module"
+	"mass" 10
+	"outfit space" -10
+	"fuel capacity" -100
+	"fuel generation" .1
+	"energy consumption" .02
+	description "To avoid the risk of being inspected by the Heliarch as they refuel on planets, Lunarium captains employ this outfit when carrying supplies or stolen outfits."
```

Comment:
```suggestion
	description "To avoid the risk of being inspected by the Heliarchs as they refuel on planets, Lunarium captains employ this outfit when carrying supplies or stolen outfits."
```